azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/configurator.py

```diff
@@ -5,23 +5,26 @@
 # --------------------------------------------------------------------------
 
 
+import logging
+
 from azure.monitor.opentelemetry.distro._diagnostics._diagnostic_logging import (
     AzureDiagnosticLogging,
 )
 from opentelemetry.sdk._configuration import _OTelSDKConfigurator
 
+_logger = logging.getLogger(__name__)
+
 
 class AzureMonitorConfigurator(_OTelSDKConfigurator):
     def _configure(self, **kwargs):
         try:
+            AzureDiagnosticLogging.enable(_logger)
             super()._configure(**kwargs)
         except ValueError as e:
-            AzureDiagnosticLogging.log_diagnostic_error(
+            _logger.error(
                 f"The components failed to initialize due to a ValueError: {e}"
```

Comment:
Can you make this more descriptive? Include "configurator" so which logger this originated from.

Reply:
Done. Also, the originating logger does show up in the diagnostic logging json